examples/nnp_training.py

```diff
@@ -156,6 +156,27 @@
 nn = torchani.ANIModel([H_network, C_network, N_network, O_network])
 print(nn)
 
+###############################################################################
+# Initialize the weights and biases.
+#
+# .. note::
+#   Pytorch default initialization for the weights and biases in linear layers
+#   is Kaiming uniform. See: `TORCH.NN.MODULES.LINEAR`_
+#   We initialize the weights similarly but from the normal distribution.
+#   The biases were initialized to zero.
+#
+# .. _TORCH.NN.MODULES.LINEAR:
+#   https://pytorch.org/docs/stable/_modules/torch/nn/modules/linear.html#Linear
+
+
+def init_params(m):
+    if isinstance(m, torch.nn.Linear):
+        torch.nn.init.kaiming_normal_(m.weight, a=math.sqrt(5))
```

Comment:
Wait, should this be a=1? The equation in https://pytorch.org/docs/stable/nn.html#torch.nn.init.kaiming_normal_ is
std=sqrt(2/(1+a^2)fan_in)...

Reply:
yes, you're right! Kaiming uniform and normal are slightly different!